mantidimaging/gui/windows/recon/view.py

```diff
@@ -81,6 +80,7 @@ def __init__(self, main_window: 'MainWindowView'):
         self.cor_table_model.rowsInserted.connect(self.on_table_row_count_change)  # type: ignore
         self.cor_table_model.rowsRemoved.connect(self.on_table_row_count_change)  # type: ignore
         self.cor_table_model.modelReset.connect(self.on_table_row_count_change)  # type: ignore
+        self.cor_table_model.dataChanged.connect(self.on_table_row_count_change)
```

Comment:
I have now ensured that on_data_change performs the fit as since these rows can change other than when a row is added, the fit needs to be ran again to ensure it is current with the data the user has entered.